### DISCUSSION TITLE
Error encountered when using local storage in Termux on a China Mobile cloudphone

### DISCUSSION THREAD

**Discussion by my208928** [AlistGo/alist#6233] [General]
I have mounted Termux on my China Mobile cloudphone, obtained root access, and created a folder named alist-files for alist, with its root path being /data/data/com.termux/files/home/alist-files. I have added local storage functionality in the backend management of alist, using the root path of alist-files. After saving and opening the frontend webpage, I encounter an issue where the page keeps loading without being accessible. Upon investigating the reason on the Termux console, I found that the problem persists as shown in the image. 
![Screenshot_20240318_022839_com chinamobile hycloudphone](https://github.com/alist-org/alist/assets/151320812/6fe34253-d609-48a9-b3c5-dc62da9bc920)
Does anyone know the cause of this issue? Your support and insights would be greatly appreciated! Thank you very much.

**Comment by njnu1115** [comment]
the error happened from this line, 
https://github.com/alist-org/alist/blob/cd5a8a011db03fbb13e002e38bcc8c08672de556/drivers/local/driver.go#L160

comment out this and the following 5 lines will workaround this issue.
